egs/aishell/s10b/local/convert_text_to_labels.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 Mobvoi AI Lab, Beijing, China (author: Fangjun Kuang)
+# Apache 2.0
+
+import argparse
+import os
+
+import kaldi
+
+
+def get_args():
+    parser = argparse.ArgumentParser(description='convert text to labels')
+
+    parser.add_argument('--lexicon-filename', dest='lexicon_filename', type=str)
+    parser.add_argument('--tokens-filename', dest='tokens_filename', type=str)
```

Comment:
Please use the standard OpenFST symbol-table format for these tokens.
I'm open to other opinions, but since we'll probably have these symbols present in FSTs I think symbol 0 should be reserved for <eps> and <blk> should be 1, and we can just apply an offset of 1 when interpreting the nnet outputs.

Reply:
... if the format is already the symbol-table format, bear in mind that the order of lines is actually arbitrary;what matters is the integer there.

Reply:
I reuse the notation from EESEN (https://github.com/srvk/eesen), which calls
`phones.txt` as `tokens.txt`.

`tokens.txt` is acutally a phone symbol table, with

```
<eps> 0
<blk> 1
other phones
```

The code here does not pose any constraint on the order of lines. What
matters here is only the integer of symbols. The first two integers `0` and `1`
are reserved. I think `0` is reserved for `<eps>`. Here I reserve `1` for
the blank symbol.

The script generating `tokens.txt` has considered the above constraint.

---

 Since there is a `T` in `TLG.fst`, I keep using `tokens.txt` here instead
of `phones.txt`.  I can switch to `phones.txt` if you think that is more natural
in kaldi.